FeedStoreChallenge/Core Data/ProductionCoreDataStack.swift

```
@@ -0,0 +1,64 @@
+
+import Foundation
+import CoreData
+
+
+public final class ProductionCoreDataStack: CoreDataStack {
+    
+    public lazy var managedContext: NSManagedObjectContext = {
+        var managedObjectContext = NSManagedObjectContext(concurrencyType: .privateQueueConcurrencyType)
+        managedObjectContext.persistentStoreCoordinator = persistentStoreCoordinator
+        return managedObjectContext
+    }()
+    
+    private lazy var persistentStoreCoordinator: NSPersistentStoreCoordinator = {
+        var coordinator = NSPersistentStoreCoordinator(managedObjectModel: managedObjectModel)
+        let storeOptions = [NSMigratePersistentStoresAutomaticallyOption : true,
+                            NSInferMappingModelAutomaticallyOption : true
+        ]
+
+        do {
+            try coordinator.addPersistentStore(ofType: NSSQLiteStoreType, configurationName: nil, at: storeURL, options: storeOptions)
+        } catch {
+            print("Unresolved error \(error)")
+            abort()
```

Errors should be handled. You can throw the error to let the clients know about issues.